### ISSUE TITLE
la generates errors with exa

### ISSUE THREAD

**Issue by scotabroad** [zpm-zsh/ls#5]
I have `exa` installed on my device running Kubuntu 20.04 LTS. However, although I can execute `ls`, `ll`, and `l`, some reason `la` generates the error "Unknown argument -A". Maybe this deals with how `ls -A` is a valid command, while `exa -A` is not, as the man page for exa only lists `-a` as a flag. Can this possibly be fixed?

**Comment by grigorii-horos** [comment]
Hm... Very strange because `-A` param present only for `ls` https://github.com/zpm-zsh/ls/blob/master/ls.plugin.zsh#L67

**Comment by scotabroad** [comment]
Okay. This is strange. So, (if I am understanding this right) since it's an `ls` param, it should not be getting used if I have `exa`, because https://github.com/zpm-zsh/ls/blob/master/ls.plugin.zsh#L24, yet somehow I still get the same error? Weird.

**Comment by grigorii-horos** [comment]
@scotabroad maybe do you have another alias/function for la?

**Comment by scotabroad** [comment]
I don't think so. My only alias in my .zshrc is `ls ="ls --color=always". Maybe one it's one of my other plugins. I'll need to take a look into that.

**Comment by scotabroad** [comment]
I couldn't find what was actually causing the problem. The only solution I found was adding `alias la="exa -a --git --icons --classify --group-directories-first --time-style=long-iso --group --color-scale"` directly to my .zshrc.

**Comment by scotabroad** [comment]
I found the issue. It was with oh-my-zsh auto-generating aliases, such as `alias la="ls -lAh"`. Disabling oh-my-zsh fixed the issue for good.
